### DISCUSSION TITLE
Without decentralized IP, complete decentralization is a joke.

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [ethereum/ethereum-org-website#11741] [Ideas]
THE IDEAL PATH
What if we replace the controlled and surveilled IP addresses with new internet addresses that are based on [CID](https://docs.ipfs.tech/concepts/content-addressing/)? These addresses will inherit all [CID](https://docs.ipfs.tech/concepts/content-addressing/)’s features, i.e., they will be purely decentralized, secure, future-proof, robust, anonymous, unhackable, controlled by no single authority and many more.

Is it just a dream? For now. If this could be true we would be changing the internet as we know it.

* [CID](https://docs.ipfs.tech/concepts/content-addressing/) address + [CID](https://docs.ipfs.tech/concepts/content-addressing/) DNS, access to Decentralized domain

win10 network adapters : Enter the [CID](https://docs.ipfs.tech/concepts/content-addressing/) address
Wi-Fi  DHCP Static : Enter the [CID](https://docs.ipfs.tech/concepts/content-addressing/) address

---
IPv4 and IPv6 are centralized. They are controlled by national companies, etc. 
![](https://www.apnic.net/wp-content/uploads/policy-environment_obsolete/images/ipv4_hierarchy.png)

Try to Google “Who controls IP addresses?” You’ll promptly get “IANA: the Internet Assigned Numbers Authority.” IANA is the top authority behind IP address allocation and assignment. There are five different regional internet registries (RIR) with jurisdiction under the IANA.

As a matter of fact, as an individual or a normal internet user you cannot request IP addresses directly from IANA or one of the five RIRs, but only from internet service providers, such as the services offered by mobile or telecom operators.

* True decentralization, no official singular Foundation, Committee, Corporation, or entities in permanent unitary control of the protocol.

**Comment by ifemimi** [comment]
> THE IDEAL PATH What if we replace the controlled and surveilled IP addresses with new internet addresses that are based on [CID](https://docs.ipfs.tech/concepts/content-addressing/)? These addresses will inherit all [CID](https://docs.ipfs.tech/concepts/content-addressing/)’s features, i.e., they will be purely decentralized, secure, future-proof, robust, anonymous, unhackable, controlled by no single authority and many more.
> 
> Is it just a dream? For now. If this could be true we would be changing the internet as we know it.
> 
> * [CID](https://docs.ipfs.tech/concepts/content-addressing/) address + [CID](https://docs.ipfs.tech/concepts/content-addressing/) DNS, access to Decentralized domain
> 
> win10 network adapters : Enter the [CID](https://docs.ipfs.tech/concepts/content-addressing/) address Wi-Fi DHCP Static : Enter the [CID](https://docs.ipfs.tech/concepts/content-addressing/) address
> 
> IPv4 and IPv6 are centralized. They are controlled by national companies, etc. ![](https://camo.githubusercontent.com/177f02b9140ba9758c0d44892cf9009a1deac408cb0e3a46157d46551459dac2/68747470733a2f2f7777772e61706e69632e6e65742f77702d636f6e74656e742f75706c6f6164732f706f6c6963792d656e7669726f6e6d656e745f6f62736f6c6574652f696d616765732f697076345f6869657261726368792e706e67)
> 
> Try to Google “Who controls IP addresses?” You’ll promptly get “IANA: the Internet Assigned Numbers Authority.” IANA is the top authority behind IP address allocation and assignment. There are five different regional internet registries (RIR) with jurisdiction under the IANA.
> 
> As a matter of fact, as an individual or a normal internet user you cannot request IP addresses directly from IANA or one of the five RIRs, but only from internet service providers, such as the services offered by mobile or telecom operators.
> 
> * True decentralization, no official singular Foundation, Committee, Corporation, or entities in permanent unitary control of the protocol.

Hey 

We’re thrilled to announce the Auto Horizon Developer Challenge, where you can build innovative AI-powered decentralized applications or agents using the powerful Autonomys Auto SDK.

🏆 Prizes:

🥇 $1,000 USDC
🥈 $500 USDC
🥉 $300 USDC
🛠 SDK Features:

Auto Drive: Decentralized storage for secure data management
Auto Consensus: PoAS layer interaction tools
Auto DAG Data: Prepare data for on-chain storage
📅 Deadline: January 31, 2025

Resources to Get Started:

[GitHub Repo](https://github.com/autonomys/auto-sdk)
Join the Community on Discord
Let’s push the boundaries of decentralized AI together! 🚀

Drop any questions below or join the Discord community for support. Let’s build something amazing! 🌟
